src/App.jsx

```diff
@@ -31,7 +32,8 @@ export default function App() {
           <Route path="/debutjeu" component={DebutJeu} />
           <Route path="/login" component={Login} />
           <Route path="/scan" component={PageScan} />
-          <Route path="/choixquestion" component={ChoixQuestion} />
+          <Route path="/choixquestion1" component={ChoixQuestion1} />
+          <Route path="/choixquestion2" component={ChoixQuestion2} />
```

Comment:
I guess you're trying to do /choixquestion/:id with:

    const {id} = useParams();

 in ChoixQuestion